analyses/focal-cn-file-preparation/01-prepare-cn-file.R

```diff
@@ -200,6 +212,30 @@ if (!dir.exists(results_dir)) {
   dir.create(results_dir)
 }
 
+# Read in GISTIC `broad_values_by_arm.txt` file
+gistic_df <-
+  data.table::fread(unzip(
+    file.path(root_dir, "data", "pbta-cnv-cnvkit-gistic.zip"),
+    files = file.path(
+      "2019-12-10-gistic-results-cnvkit",
```

Comment:
I have a thing I wrote to avoid this dated folder in case it gets updated. Unsure if it is completely necessary, but here: 
 ```
# Path to gistic zip file results
gistic_zip <- file.path(input_dir, "pbta-cnv-cnvkit-gistic.zip")

# Get the name of the folder first since this changes with updates to the date
folder_name <- unzip(zipfile = gistic_zip, list = TRUE) %>%
  dplyr::filter(Length == 0) %>%
  dplyr::pull("Name")

# Extract files if it hasn't been done yet
if (!dir.exists(gistic_results)) {
  # Unzip but only extract the files not the folder
  unzip(
    zipfile = gistic_zip,
    exdir = gistic_results
  )
}
# Designate GISTIC results folder to extract to
gistic_results <- file.path(gistic_results, folder_name)
```